knowledge-base/how-to-fix-reportscontrollerbase-is-obsolete.md

```diff
@@ -0,0 +1,48 @@
+---
+title: How to fix: ReportsControllerBase is obsolete
+description: Such error can appear in .NET Core application after upgrade
+type: troubleshooting
+page_title: How to Fix: ReportsControllerBase is Obsolete
+slug: how-to-fix-reportscontrollerbase-is-obsolete
+position: 
+tags: 
+ticketid: 1462359
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product Version</td>
+			<td>14.0.20.115+</td>
+		</tr>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+In [R1 2020](https://www.telerik.com/support/whats-new/reporting/release-history/progress-telerik-reporting-r1-2020-14-0-20-115) we made some improvements and now we inherit the [ReportsControllerBase](../t-telerik-reporting-services-webapi-reportscontrollerbase) type and inject the configuration settings in the constructor. 
```

Comment:
Consider changing 'and now we inherit...' to something like 'and now you can use Dependency Injection and pass the configuration settings in the constructor of the ReportsController'.